src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -414,4 +422,20 @@ public ResponseEntity<Void> deleteExpiredArtefacts() {
         publicationService.deleteExpiredArtefacts();
         return ResponseEntity.noContent().build();
     }
+
+    @ApiResponses({
+        @ApiResponse(responseCode = OK_CODE, description = "Artefact of ID {} has been archived"),
+        @ApiResponse(responseCode = AUTH_ERROR_CODE, description = UNAUTHORIZED_DESCRIPTION),
+        @ApiResponse(responseCode = NOT_FOUND_CODE, description = "Artefact of ID {} not found when archiving")
+    })
+    @Operation(summary = "Delete all expired artefacts")
```

Comment:
Can we rename the description to "archive artefact rather than delete artefact pls? :)

Reply:
Good spot! Updated :)